simpletransformers/t5/t5_model.py

```diff
@@ -91,11 +91,18 @@ def __init__(
 
         self.results = {}
 
-        self.config = T5Config.from_pretrained(model_name, **self.args.config)
-
-        self.model = T5ForConditionalGeneration.from_pretrained(model_name, config=self.config)
+        if model_name==None:
```

Comment:
```suggestion
        if model_name is None:
```

It will work fine without this change but this is the recommended best practice.